src/plane.rs

```diff
@@ -306,23 +315,64 @@ impl<'a, T: Pixel> ExactSizeIterator for IterWidth<'a, T> { }
 
 impl<'a, T: Pixel> FusedIterator for IterWidth<'a, T> { }
 
+pub struct RowsIter<'a, T: Pixel> {
+  ps: &'a PlaneSlice<'a, T>,
+  next_row: usize,
+}
+
+impl<'a, T: Pixel> Iterator for RowsIter<'a, T> {
+  type Item = &'a [T];
+
+  fn next(&mut self) -> Option<Self::Item> {
+    let remaining = self.ps.plane.cfg.height as isize - self.ps.y + self.next_row as isize;
```

Comment:
Oops, parentheses missing:

```rust
let remaining = self.ps.plane.cfg.height as isize - (self.ps.y + self.next_row as isize);
```

It's fixed in my `PlaneMutSlice` pull request #1040.